product_multi_barcode_constraint_per_company/tests/test_product_barcode_constraint.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2 import IntegrityError
+
+from odoo import _
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+from odoo.tools.misc import mute_logger
+
+from odoo.addons.product_barcode_constraint_per_company.tests.common import (
+    CommonProductBarcodeConstraintPerCompany,
+)
+
+
+@tagged("at_install")
```

Comment:
**chore**: same as above, this is the default, this line is not necessary

Reply:
I'll fix it